sql/core/src/test/scala/org/apache/spark/sql/errors/QueryCompilationErrorsSuite.scala

```diff
@@ -616,6 +622,31 @@ class QueryCompilationErrorsSuite
         "functionName" -> "`array_contains`",
         "classCanonicalName" -> "org.apache.spark.sql.catalyst.expressions.ArrayContains"))
   }
+
+  test("RENAME_SRC_PATH_NOT_FOUND: rename the file which source path does not exist") {
```

Comment:
Could you move the test from `*Compilation*Suite` to `QueryExecutionErrorsSuite`, please.

Reply:
@MaxGekk done.